{
    "ID": {
        "GitHub_Advisory_Database": {
            "aliases": [
                "CVE-2024-39614"
            ]
        },
        "Nomisec": {
            "id": 837370546,
            "name": "-CVE-2024-39614",
            "full_name": "Abdurahmon3236/-CVE-2024-39614"
        }
    },
    "summary": "Django vulnerable to Denial of Service",
    "details": {
        "GitHub_Advisory_Database": "An issue was discovered in Django 5.0 before 5.0.7 and 4.2 before 4.2.14. `get_supported_language_variant()` was subject to a potential denial-of-service attack when used with very long strings containing specific characters."
    },
    "description": {
        "Nomisec": null
    },
    "topics": [
        "characters",
        "potential",
        "long",
        "containing",
        "specific",
        "14",
        "attack",
        "denial",
        "used",
        "discovered"
    ],
    "severity": [],
    "affected": [],
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://nvd.nist.gov/vuln/detail/CVE-2024-39614"
        },
        {
            "type": "WEB",
            "url": "https://github.com/django/django/commit/17358fb35fb7217423d4c4877ccb6d1a3a40b1c3"
        },
        {
            "type": "WEB",
            "url": "https://github.com/django/django/commit/8e7a44e4bec0f11474699c3111a5e0a45afe7f49"
        },
        {
            "type": "WEB",
            "url": "https://docs.djangoproject.com/en/dev/releases/security"
        },
        {
            "type": "PACKAGE",
            "url": "https://github.com/django/django"
        },
        {
            "type": "WEB",
            "url": "https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2024-59.yaml"
        },
        {
            "type": "WEB",
            "url": "https://groups.google.com/forum/#%21forum/django-announce"
        },
        {
            "type": "WEB",
            "url": "https://www.djangoproject.com/weblog/2024/jul/09/security-releases"
        }
    ],
    "proof_of_concept": {
        "owner": {
            "html_url": "https://github.com/Abdurahmon3236"
        },
        "html_url": "https://github.com/Abdurahmon3236/-CVE-2024-39614"
    },
    "has_discussions": null,
    "database_specific": {
        "cwe_ids": [
            "CWE-130",
            "CWE-770"
        ],
        "severity": "HIGH",
        "github_reviewed": true,
        "github_reviewed_at": "2024-07-10T21:43:25Z",
        "nvd_published_at": "2024-07-10T05:15:12Z"
    },
    "Time_Description": {
        "GitHub_Advisory_Database": {
            "modified": "2024-08-02T15:52:17Z",
            "published": "2024-07-10T06:33:52Z",
            "withdrawn": null
        },
        "Nomisec": {
            "created_at": "2024-08-02T20:11:29Z",
            "updated_at": "2024-08-29T13:31:15Z",
            "pushed_at": "2024-08-02T20:15:39Z"
        }
    }
}